pkg/process/metadata/workloadmeta/grpc_test.go

```diff
@@ -47,6 +49,7 @@ func TestGetGRPCStreamPort(t *testing.T) {
 
 func TestStartStop(t *testing.T) {
 	cfg := config.Mock(t)
+	fxutil.Test[telemetry.Mock](t, telemetry.MockModule).Reset()
```

Comment:
Just making sure, does this auto cleanup?

Reply:
[Yep!](https://github.com/DataDog/datadog-agent/blob/e30cf11eb10276dcf3e6ffbfa43ce5c7531360a7/pkg/util/fxutil/test.go#L40)